packages/gatsby-admin/src/components/navbar.tsx

```diff
@@ -24,31 +35,34 @@ function Navbar(): JSX.Element {
         paddingY: 5,
       }}
     >
-      <Flex gap={5} alignItems="center">
-        <Text>Gatsby Admin</Text>
-        {data && data.npmPackageJson && (
-          <div
-            sx={{
-              width: `1px`,
-              height: `16px`,
-              backgroundColor: `blackFade.50`,
-            }}
-          />
-        )}
+      <Flex gap={5} alignItems="baseline">
+        <Text sx={{ textTransform: `uppercase`, fontSize: 0 }}>
+          Gatsby Admin
+        </Text>
         {data && data.npmPackageJson && (
-          <Text sx={{ fontWeight: `bold`, color: `text.primary` }}>
+          <Text sx={{ fontWeight: `bold`, color: `text.primary`, fontSize: 3 }}>
             {data.npmPackageJson.value.replace(/^"|"$/g, ``)}
           </Text>
         )}
       </Flex>
-      <Flex alignItems="center">
-        <BaseAnchor
-          href={`/`}
+      <Flex alignItems="baseline" gap={3}>
+        <FeedbackForm
+          projectId="9502a819990b03"
+          triggerComponent={SendFeedbackButton}
+        />
+        <AnchorButton
+          size="S"
+          href="/___graphql"
           target="_blank"
-          sx={{ color: `blackFade.60`, textDecoration: `none` }}
+          variant="SECONDARY"
         >
-          Visit site
-        </BaseAnchor>
+          GraphQL&nbsp;
```

Comment:
Maybe even say `GraphiQL` ?

Reply:
Ya, I'd agree with this. Probably a conversation for design, but we do use this term in the docs and it would be good to stay consistent.